src/_premake_init.lua

```diff
@@ -1062,6 +1062,13 @@
 		kind = "list:table",
 	}
 
+	api.register {
+		name = "props",
```

Comment:
I wonder if the name should not reflect that it is visual specific (`vsprops`)?

Reply:
Renamed to vsprops

Reply:
Late to the party, but can this life in `modules/vstudio/_preload.lua` instead, since it's specific to VS?

Reply:
Thanks for pointing this out. I have moved the definition to `modules/vstudio/_preload.lua`.